prod-eu-west/vpc/waf.tf

```diff
@@ -214,12 +68,134 @@ resource "aws_wafv2_web_acl" "prod-default" {
+                search_string = "bearer "
+                field_to_match {
+                  single_header {
+                    name = "authorization"
+                  }
+                }
+                positional_constraint = "STARTS_WITH"
+                text_transformation {
+                  priority = 0
+                  type     = "LOWERCASE"
+                }
+              }
+            }
+            statement {
+              byte_match_statement {
+                search_string = "basic "
+                field_to_match {
+                  single_header {
+                    name = "authorization"
+                  }
+                }
+                positional_constraint = "STARTS_WITH"
+                text_transformation {
+                  priority = 0
+                  type     = "LOWERCASE"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+
+    visibility_config {
+      cloudwatch_metrics_enabled = true
+      metric_name                = "prodRateLimitingRuleAuthenticated"
+      sampled_requests_enabled   = true
+    }
+  }
+
+   rule {
+    name     = "prodRateLimitWithEmail"
+    priority = 3
+
+    action {
+      block {
+        custom_response {
+          response_code = 403
+          custom_response_body_key = "ratelimiterror"
+        }
+      }
+    }
+
+    statement {
+      rate_based_statement {
+        limit              = 1000
+        aggregate_key_type = "IP"
+
+        scope_down_statement {
+          or_statement {
+            statement {
+              byte_match_statement {
+                search_string = "@"
+                field_to_match {
+                  single_header {
+                    name = "user-agent"
+                  }
+                }
+                positional_constraint = "CONTAINS"
+                text_transformation {
+                  priority = 0
+                  type     = "NONE"
+                }
+              }
+            }
+            statement {
+              byte_match_statement {
+                search_string = "mailto="
+                field_to_match {
+                  query_string {}
+                }
+                positional_constraint = "CONTAINS"
+                text_transformation {
+                  priority = 0
+                  type     = "NONE"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+
+    visibility_config {
+      cloudwatch_metrics_enabled = true
+      metric_name                = "prodRateLimitWithEmail"
+      sampled_requests_enabled   = true
+    }
+  }
+
+  rule {
+    name     = "prodRateLimitUnauthenticated"
+    priority = 4
+
+    action {
+      block {
+        custom_response {
+          response_code = 403
+          custom_response_body_key = "ratelimiterror"
+        }
+      }
+    }
+
+    statement {
+      rate_based_statement {
+        limit              = 500
```

Comment:
This seems rather low actually, is this intentional? 3,000 is the current general limit.

Reply:
It is intentional https://github.com/datacite/product-backlog/issues/374, maybe @codycooperross can weigh in.

Reply:
Confirming that these are the numbers that have been in place for some time, yes. We communicated these rates to the community for the past few months. Here is the proposal document with some usage analysis: https://docs.google.com/document/d/1Cc0QuSZzpyQqo_KwSuXYv-pI8e6wQTOPEwvrMcXU6z4/edit?tab=t.0